bin/bundle

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env ruby2.1
```

Comment:
I actually prefer just use <code>ruby</code>.. cause then we no need to maintain here once we updated the ruby version.

And If we restrict to specific version, without that version user cannot run right ?